contracts/ReadBytesAt.sol

```diff
@@ -0,0 +1,13 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+contract ReadBytesAt {
+    // NOTE: pure seems like the wrong mutability annotation
```

Comment:
I'd agree. This should be `view` in my opinion, and should perhaps be commented to note that is unsafe in its assumption that there is a `bytes` array at the specified slot.

Reply:
It's an error in forge/solc unless it's `pure`, hence the comment.

Reply:
I've updated the comment.